js/src/edit.js

```diff
@@ -23,6 +23,40 @@ if( window.wpseoPostScraperL10n ) {
 	localizedData = wpseoTermScraperL10n;
 }
 
+const Icon = () => (
+	<svg fill="#333" version="1.1" id="Layer_2_1_" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink" viewBox="0 0 841.9 595.3" enableBackground="new 0 0 841.9 595.3" xmlSpace="preserve">
```

Comment:
It would be nice to use the attribute we usually add to SVG icons for a11y: ` role="img" aria-hidden="true" focusable="false"`